src/components/checkout.js

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+const buttonStyles = {
+  fontSize: "16px",
+  textAlign: "center",
+  color: "#fff",
+  outline: "none",
+  padding: "12px 60px",
+  boxShadow: "2px 5px 10px rgba(0,0,0,.1)",
+  backgroundColor: "rgb(255,0,0)",
+  borderRadius: "6px",
+  letterSpacing: "1.5px",
+}
+const Checkout = class extends React.Component {
+  // Initialise Stripe.js with your publishable key.
+  // You can find your key in the Dashboard:
+  // https://dashboard.stripe.com/account/apikeys
+  componentDidMount() {
```

Comment:
You can do
```
if(!window) return
this.stripe = window.Stripe("pk_test_XiirziufElakjoOpyuyCrPfo")
``` 
If there is no window it will no continue to the next line. This is a pattern named fail early that I like.

You can also do
```
if(window) {
  this.stripe = window.Stripe("pk_test_XiirziufElakjoOpyuyCrPfo")
}
```
no need for a else when you do not want to do anything on else.

Reply:
Ok I use

if(!window) return
this.stripe = window.Stripe("pk_test_XiirziufElakjoOpyuyCrPfo")